GeneralsMD/Code/GameEngine/Include/GameClient/GameWindowTransitions.h

```diff
@@ -671,18 +676,20 @@ class GameWindowTransitionsHandler: public SubsystemInterface
 	static const FieldParse m_gameWindowTransitionsFieldParseTable[];																				///< the parse table
 	static void parseWindow( INI* ini, void *instance, void *store, const void *userData );
 
-	void setGroup(AsciiString groupName, Bool immidiate = FALSE);		// THis will be the next group to fire off.
+	void setGroup(AsciiString groupName, Bool immediate = FALSE, Bool skip = FALSE); // This will be the next group to fire off.
```

Comment:
I think we can get rid of skip parameter by allowing to access TransitionGroup pointer.